teuthology/task/tests/__init__.py

```diff
@@ -44,8 +44,10 @@ def pytest_generate_tests(self, metafunc):
             metafunc.parametrize(["ctx", "config"], [(self.ctx, self.config),])
 
     # log the outcome of each test
-    def pytest_runtest_makereport(self, __multicall__, item, call):
-        report = __multicall__.execute()
+    @pytest.hookimpl(hookwrapper=True)
+    def pytest_runtest_makereport(self, item, call):
+        outcome = yield
+        report = outcome.get_result()
 
         # after the test has been called, get it's report and log it
```

Comment:
can we fix that "it's" while we're here.  grinds my gears.

Reply:
done